### PR TITLE
[Blog] Yash's journey to GSoC  @Layer5

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by willcalcote** [comment]
Screen Shot 2022-06-09 at 12 23 59 PM
here is a .png file on what it looks like so far, do you have any feedback on it?

File: src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "My Journey as a Contributor to GSoC Intern @Layer5"
+subtitle: "A Rollercoaster experience"
+date: 2022-06-10 08:00:00 -0630
+author: Yash Kamboj
+thumbnail: 
+category: Community
+description: "Aaditya Narayan Subedy's journey from a Contributor @Layer5 to getting an GSoC internship @Layer5"
```
Comment (Abhishek-kumar09) [inline src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx:8]: Templates are fun

Comment (YashKamboj) [reply src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx]: Missed that 😅😅

Comment (YashKamboj) [reply src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx]: @Abhishek-kumar09 Thank you for pointing that out  😅

**Pull request closed without merging**